contracts/AssetToken.sol

```diff
@@ -24,10 +27,14 @@ contract AssetToken is IERC777, Initializable {
     address private _owner;
     address private _emergencyDelegate;
     address private _fundingDelegate;
+    address private _listsController;
     bool private _isActive;
 
     mapping(address => uint256) private _balances;
 
+    mapping(address => bool) private _isBlacklisted;
```

Comment:
I don't think we should mix blacklist/whitelist and allow/deny in the code. It does come across as slightly confusing